tests/bluetooth/test_gatt_connection.py

```diff
@@ -0,0 +1,67 @@
+#  Copyright 2025 Google LLC.
+#  Copyright (c) Microsoft Corporation.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+import pytest_asyncio
+from test_helpers import (execute_command, send_JSON_command, subscribe,
+                          wait_for_event)
+
+from . import disable_simulation, setup_granted_device
+
+
+@pytest_asyncio.fixture(autouse=True)
```

Comment:
Can this fixture be moved to `__init__.py`?

Reply:
I tried, but it doesn't take effect. I did some searching and it seems to be suggested to have the fixture inside the test file rather than inside `__init__.py`.